SweetsFactory/src/app/app.module.ts

```diff
@@ -1,23 +1,35 @@
 import { NgModule } from '@angular/core';
-import { BrowserModule }  from '@angular/platform-browser';
+import { BrowserModule } from '@angular/platform-browser';
 import { AppComponent } from './start/app.component';
+import { FormsModule } from '@angular/forms';

```

Comment:
Is there a place where you use this at this point?

Reply:
I have to import Forms module so that I could put @Input() decorator form the MenuItemComponent so that I could send them as parameters in the navbar template